core/src/main/resources/sql-translate.properties

```diff
@@ -0,0 +1,103 @@
+nextval.source.Oracle=(\w+)\.NEXTVAL
+nextval.target.MS_SQL=NEXT VALUE FOR $1
+nextval.target.MySql =NULL
+
+currval.source.Oracle=(\w+)\.CURRVAL
+currval.target.MS_SQL=(SELECT current_value FROM sys.sequences WHERE name = '$1')
+currval.target.MySql =NULL
+
```

Comment:
Misschien is het handig boven aan dit bestand wat uitleg te geven hoe de opbouw in elkaar zit. Je mag comments (met `#` prefix) gebruiken.